src/main.rs

```diff
@@ -14,34 +16,56 @@ fn main() {
         .version(env!("CARGO_PKG_VERSION"))
         .author(env!("CARGO_PKG_AUTHORS"))
         .about(env!("CARGO_PKG_DESCRIPTION"))
-        .arg(Arg::with_name("query")
-            .short("q")
-            .required(true)
-            .index(1))
+        .arg(Arg::with_name("query").short("q").required(true).index(1))
         .get_matches();
 
+    let mut easy_client = Easy::new();
+    easy_client
+        .useragent("Alfred Crates (robjtede fork)")
```

Comment:
lol, please dont leave this user agent in as it is

```suggestion
        .useragent(format!("{} (v{})", env!("CARGO_PKG_name", env!("CARGO_PKG_VERSION"))))
```

Reply:
Updated in a new commit.